doc/Manuals/ig-ss_x-road_v6_security_server_installation_guide_for_rhel7.md

```diff
@@ -0,0 +1,297 @@
+ &nbsp; | TCP 5577                                | Port for inbound connections (from the external network to the security server)<br> Querying of OCSP responses between security servers
+ &nbsp; | TCP 2080                                | Port for inbound connections (from the external network to the security server)<br> Message exchange between security server and operational data monitoring daemon (by default on localhost)
+ &nbsp; | TCP 9011                                | Port for inbound connections (from the external network to the security server)<br> Operational data monitoring daemon JMX listening port
+  &nbsp; | TCP 9999                                | Port for inbound connections (from the external network to the security server)<br> Environmental monitoring daemon JMX listening port
+ 1.5  | TCP 5500                                  | Ports for outbound connections (from the security server to the external network)<br> Message exchange between security servers
+ &nbsp; | TCP 5577                                | Ports for outbound connections (from the security server to the external network)<br> Querying of OCSP responses between security servers
+ &nbsp; | TCP 4001                                | Ports for outbound connections (from the security server to the external network)<br> Communication with the central server
+ &nbsp; | TCP 80                                  | Ports for outbound connections (from the security server to the external network)<br> Downloading global configuration
+ &nbsp; | TCP 80,443                              | Ports for outbound connections (from the security server to the external network)<br> Most common OCSP and time-stamping services
+ 1.6  | TCP 4000                                  | User interface (local network)
+ 1.7  | TCP 8080                                  | Information system access points (in the local network)<br> Connections from information systems
+ &nbsp; | TCP 8443                                | Information system access points (in the local network)<br> Connections from information systems
+ 1.8  |                                           | Security server internal IP address(es) and hostname(s)
+ 1.9  |                                           | Security server public IP address, NAT address
+
+
+### 2.3 Requirements for the Security Server
+
+Minimum recommended hardware parameters:
+
+-   the server’s hardware (motherboard, CPU, network interface cards, storage system) must be supported by RHEL7 in general;
+
+-   a 64-bit dual-core Intel, AMD or compatible CPU; AES instruction set support is highly recommended;
+
+-   2 CPU;
+
+-   4 GB RAM;
+
+-   10 GB free disk space (OS partition), 20-40 GB free disk space (`/var` partition);
+
+-   a 100 Mbps network interface card;
+
+-   if necessary, interfaces for the use of hardware tokens.
```

Comment:
section 2.7 mentions that no support for hw tokens on rhel 7 is provided, maybe this line should be removed